handout/sections/header.tex

```diff
@@ -128,6 +135,9 @@
   {\par\normalfont\hfill--\ \chapquote@author\hspace*{\@tempdima}\par\bigskip}
 \makeatother
 
+% OSE font settings
```

Comment:
Can we get rid fo this? and the other file?

Reply:
Removed the extra preamble and any font-settings that require .sty files.